### PR TITLE
Update upgrade command for 2024.7.1

### PR DESCRIPTION
**Pull request by viniciusdc**
## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [ ] Did you test the pull request locally?
- [ ] Did you add new tests?

### CODE REVIEW

File: src/_nebari/upgrade.py [hunk 1]
```diff
@@ -1182,6 +1182,29 @@ def _version_specific_upgrade(
...
+        - Digital Ocean deprecation warning.
+    """
+
+    version = "2024.7.1"
+
+    @override
+    def _version_specific_upgrade(
+        self, config, start_version, config_filename: Path, *args, **kwargs
+    ):
+        if config.get("provider", "") == ProviderEnum.gcp.value:
```
Comment (marcelovilla) [inline src/_nebari/upgrade.py:1199]: We want to have `do` instead of `gcp` here

**Review by marcelovilla — CHANGES_REQUESTED** [review]

File: src/_nebari/upgrade.py (hunk 1 above)
Comment (viniciusdc) [reply src/_nebari/upgrade.py:1199]: ops, my copy and paste failed me

Comment (viniciusdc) [reply src/_nebari/upgrade.py]: done

**Review by marcelovilla — APPROVED** [review]
Thanks @viniciusdc !

**Review by Adam-D-Lewis — APPROVED** [review]

**Pull request merged**